docs/user-guide/federation/deployment.md

```diff
@@ -0,0 +1,109 @@
+---
+---
+
+This guide explains how to use Deployments in the Federation control plane.
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+This guide assumes that you have a running Kubernetes Cluster
+Federation installation. If not, then head over to the
+[federation admin guide](/docs/admin/federation/) to learn how to
+bring up a cluster federation (or have your cluster administrator do
+this for you). Other tutorials, for example
+[this one](https://github.com/kelseyhightower/kubernetes-cluster-federation)
+by Kelsey Hightower, are also available to help you.
+
+You are also expected to have a basic
+[working knowledge of Kubernetes](/docs/getting-started-guides/) in
+general and [ReplicaSets](/docs/user-guide/replicasets/) in particular.
+
+## Overview
+
+Deployments in federation control plane (referred to as "federated Deployments" in
+this guide) are very similar to the traditional [Kubernetes
+ReplicaSets](/docs/user-guide/replicasets/), and provide the same functionality.
+Creating them in the federation control plane ensures that the desired number of
+replicas exist across the registered clusters.
+
+** Federated Deployment is currently in ALPHA, which means that the core functionality
```

Comment:
If your intention is to make this bold, then I think you should remove the whitespace between ** Federated and yet. **